examples/hello_world.rs

```diff
@@ -55,28 +55,26 @@ pub fn main() {
         println!("accept error = {:?}", err);
     });
 
+    println!("server running on localhost:6142");
+
     // This starts the `current_thread` executor.
     //
     // Executors are responsible for scheduling many asynchronous tasks, driving
     // them to completion. There are a number of different executor
     // implementations, each providing different scheduling characteristics.
     //
     // The `current_thread` executor multiplexes all scheduled tasks on the
-    // current thread. This means that spawned tasks are not required to
-    // implement `Send`.
-    current_thread::run(|_| {
-        // Now, the server task must be spawned.
-        //
-        // It's important to note that all futures / tasks are lazy. No work
-        // will happen unless they are spawned onto an executor.
-        current_thread::spawn(server);
-
-        println!("server running on localhost:6142");
-
-        // The `current_thread::run` function will now block until *all* spawned
-        // tasks complete.
-        //
-        // In our example, we have not defined a shutdown strategy, so
-        // this will block until `ctrl-c` is pressed at the terminal.
-    });
+    // current thread. This means that spawned tasks must not implement `Send`.
```

Comment:
Wasn't the old wording correct - "spawned tasks are not required to implement `Send`"? :)

Reply:
You are correct!

Reply:
Well, the good news is it has been fixed on master it looks like.